src/table.js

```diff
@@ -164,8 +173,9 @@ export function Table(
   }
 
   root.onscroll = () => {
-    if (root.scrollHeight - root.scrollTop < 400 && n < data.length) {
+    while (root.scrollHeight - root.scrollTop - spacer.clientHeight < 400 && n < data.length) {
```

Comment:
If you scroll really fast, you’ll end up below the newly added rows after one pass, so we keep looping until in-view rows are rendered.